### PR TITLE
add mode option

### PR DESCRIPTION
**Pull request by kasperpeulen**
I'm using this fork locally, it was useful for me, so I thought maybe useful for others as well.

### CODE REVIEW

File: bin/peanut.dart [hunk 1]
```diff
@@ -12,6 +12,7 @@ void main(List<String> args) {
   ArgParser parser = new ArgParser(allowTrailingOptions: false)
     ..addOption(_directoryFlag, abbr: 'd', defaultsTo: 'web')
     ..addOption('branch', abbr: 'b', defaultsTo: 'gh-pages')
+    ..addOption('mode', defaultsTo: 'release')
```
Comment (kevmoo) [inline bin/peanut.dart]: Update this to include only the valid options: I think it's `release` and `debug`? Or dev? Whatever it is.

**Comment by kevmoo** [comment]
Please update the version to 0.2.0 and update the changelog.

Thanks!

**Comment by kasperpeulen** [comment]
@kevmoo done

**Comment by kevmoo** [comment]
And to encourage best-practices: would you `rebase -i` and squish your commits?

**Comment by kasperpeulen** [comment]
@kevmoo like this?

**Comment by kevmoo** [comment]
:metal: – will get it in after lunch!

**Comment by kevmoo** [comment]
Thanks! Closed with 42afc89eea2906bac77b4a2d90595f075c31ad4c

**Pull request closed without merging**